common/src/main/java/org/apache/uniffle/common/compression/Lz4Codec.java

```diff
@@ -38,4 +40,22 @@ public void decompress(ByteBuffer src, int uncompressedLen, ByteBuffer dest, int
   public byte[] compress(byte[] src) {
     return lz4Factory.fastCompressor().compress(src);
   }
+
+  @Override
+  public int compress(ByteBuffer src, ByteBuffer dest) {
+    try {
+      final int cpLen = lz4Factory.fastCompressor()
```

Comment:
lz4 have the method.
```
  public final void compress(ByteBuffer src, ByteBuffer dest) {
    int cpLen = this.compress(src, src.position(), src.remaining(), dest, dest.position(), dest.remaining());
    src.position(src.limit());
    dest.position(dest.position() + cpLen);
  }
```
Why don't use it?

Reply:
my mistake. I made a mistake of dest position in my previous test for this function, just now I test it is ok.